src/main/java/com/lotto/domain/LotteryNumber.java

```diff
@@ -0,0 +1,54 @@
+package com.lotto.domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LotteryNumber implements Comparable<LotteryNumber> {
+
+    private static final int START_NUMBER = 1;
+    private static final int END_NUMBER = 45;
+    private static final int NUMBER_COUNT = 6;
+    private static final List<LotteryNumber> LOTTERY_NUMBER_LIST = IntStream.range(START_NUMBER, START_NUMBER + END_NUMBER)
```

Comment:
```suggestion
    private static final List<LotteryNumber> LOTTERY_NUMBER_LIST = IntStream.rangeClosed(START_NUMBER, END_NUMBER)
```
위와 같이 선언할 수도 있을것 같습니다 :)